dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/DimensionsCriteria.java

```diff
@@ -38,5 +40,5 @@
 @NoArgsConstructor
 public class DimensionsCriteria extends PagingAndSortingCriteriaAdapter
 {
-    private String filter;
+    private Collection<String> filter;
```

Comment:
I'm wondering if repeated filters should be allowed...
If they are not allowed, we could:
1) Use Set
2) Or add validation on `filter`

Reply:
good point